posts/2024-05-13-jakarta-nosql-challenge-switching-nosql-easily.adoc

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+title: "Jakarta NoSQL in Action: Switching NoSQL Databases with Ease"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/dearrudam
+author_github: https://github.com/dearrudam
+seo-title: TITLE - OpenLiberty.io
+seo-description: |
```

Comment:
this pipe is causing a metadata error that will prevent the post from building- need to delete it and bring the description text up a line